test/vm.cpp

```diff
@@ -513,13 +513,39 @@ BOOST_AUTO_TEST_CASE(vmSystemOperationsTest)
 	dev::test::executeTests("vmSystemOperationsTest", "/VMTests", dev::test::doVMTests);
 }
 
+BOOST_AUTO_TEST_CASE(vmPerformanceTest)
+{
+	for (int i = 1; i < boost::unit_test::framework::master_test_suite().argc; ++i)
+	{
+		string arg = boost::unit_test::framework::master_test_suite().argv[i];
+		if (arg == "--performance")
+		{
+			auto start = chrono::steady_clock::now();
+
+			dev::test::executeTests("vmPerformanceTest", "/VMTests", dev::test::doVMTests);
+
+			auto end = chrono::steady_clock::now();
+			chrono::milliseconds duration(chrono::duration_cast<chrono::milliseconds>(end - start));
```

Comment:
I see your point, so you think it is more readable, and you can switch precision at on place, not two. (I just dont like auto too much, it feels so python :-)  ). For me its more readable when I see the type directly. But I will change it to auto in order to have only one place to touch when I would change precision.